### DISCUSSION TITLE
Allow enlarging SVG when font size is changed

### DISCUSSION THREAD

**Discussion by llemeurfr** [edrlab/thorium-reader#1743] [Ideas]
The gray circle remains the same size when the font size is changed. There is no obvious method to increase the size of the SVG image within the reader. (R.Orme testing with an sample EPUB).

**Comment by llemeurfr** [comment]
The test book is in http://epubtest.org/testbooks, section visual adjustment tests. The test is visual-710 Enlarge SVG image.

**Comment by llemeurfr** [comment]
http://epubtest.org/testbooks does not exist anymore.

**Comment by danielweck** [comment]
Some SVG can indeed appear in the text flow for typographic purpose, other SVG is more similar to image illustrations, diagrams, etc.
To discuss...
